application/tests/librarian/evaluate_harness_test.py

```
@@ -0,0 +1,251 @@
+        ]
+        pipe = CountingPipeline({"alpha": [("616-305", 4.0)], "beta": [("111-111", 3.0)]})
+
+        audits = harness.live_audits(rows, pipe, pipe)
+
+        self.assertEqual(pipe.retrieve_calls, 2)
+        self.assertEqual(pipe.rerank_calls, 2)
+        self.assertEqual(set(audits), {"p1", "n1"})
+
+        # Three reports read the same audits; none of them may re-run the pipeline.
+        harness.report_retrieval_recall(rows, audits, 10, 5)
+        status, scaler = harness.report_calibration(rows, audits)
```

Ruff (0.16.0)

[warning] Unpacked variable `status` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)